### ISSUE TITLE
form submit error

### ISSUE THREAD

**Issue by devHudson** [yabwe/medium-editor#852]
If there is no keypress operation in the editor, it forms submit when the value does not exist (textarea)
To the situation after submit
Example 1) Image uploading only if, there's no tag.
Example 2) First, the article without image tag, top image and write only exist.
Example 3) to put the articles stored in the db to form, if you submit without modification, the tag does not come across articles
Given this situation, action must be unconditional keypress html tag is taken over

please help me

**Comment by nmielnik** [comment]
Sorry @devHudson I'm having trouble understanding your question, could you rephrase the question or try to provide a jsfiddle or codepen which demonstrates the issue?
